src/components/pages/Offerers/OfferersContainer.jsx

```diff
@@ -1,21 +1,64 @@
 import { connect } from 'react-redux'
 import { compose } from 'redux'
 
-import {
-  withFrenchQueryRouter,
-  withRedirectToSigninWhenNotAuthenticated,
-} from 'components/hocs'
+import { withFrenchQueryRouter, withRedirectToSigninWhenNotAuthenticated } from 'components/hocs'
 import Offerers from './Offerers'
+import {closeNotification, showNotification} from 'pass-culture-shared'
+import {assignData, requestData} from 'redux-saga-data'
+import { offererNormalizer } from '../../../utils/normalizers'
 
 export const mapStateToProps = (state) => {
   return {
     pendingOfferers: state.data.pendingOfferers,
     offerers: state.data.offerers,
+    notification: state.notification,
   }
 }
 
+export const mapDispatchToProps = (dispatch) => ({
+  assignData:() => dispatch(assignData({ offerers: [], pendingOfferers: [] })),
+
+  closeNotification:() => dispatch(closeNotification()),
+
+  loadOfferers: (apiPath, handleFail, handleSuccess) => () => {
+    dispatch(
+      requestData({
+        apiPath,
+        handleFail: handleFail,
+        handleSuccess: handleSuccess,
+        normalizer: offererNormalizer,
+      })
+    )
+  },
+
+  loadNotValidatedUserOfferers: (notValidatedUserOfferersPath) => () => {
```

Comment:
```suggestion
  loadNotValidatedUserOfferers: apiPath => () => {
```
L'ancien paramètre est trop redondant avec le nom de la fonction et puis tu utilises déjà ce paramètre juste au dessus, autant est raccord.